### PR TITLE
Benchmark/nbody

### PR DESCRIPTION
**Pull request by IR0NSIGHT**

### CODE REVIEW

**Comment by IR0NSIGHT** [comment]
fixes #9

File: src/nbody.effekt [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+def nbody(problemSize: Int) = {
+  val PI = 3.141592653589793
+  val SOLAR_MASS = 4.0 * PI * PI
+  val DAYS_PER_YER = 365.24;
+
+  def offsetMomentum(px: Double, py: Double, pz: Double, body: Body) = {
+    Body(
+      body.x, body.y, body.z,
+      0.0 -(px / SOLAR_MASS),
```
Comment (phischu) [inline src/nbody.effekt:36]: We should add a definition `def neg(x: Double): Double` to the standard library. Likewise for the bitshift operations you've added in another benchmark. Perhaps you could make a pull request on the effekt repo?

File: src/nbody.effekt [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+    energy(bodies)
+  }
+
+  def verifyResult(result: Double): Boolean = {
+     if (problemSize == 250000) {
+       (result - 0.1690859889909308) < 0.0000000000000001;
+      } else if (problemSize == 1) {
+        result == -0.16907495402506745;
+      }  else {
+        return false;
```
Comment (phischu) [inline src/nbody.effekt:192]:
```suggestion
        false
```

**Review by phischu — APPROVED** [review]
Excellent!

**Pull request merged**